Middleware/src/test/java/ca/concordia/ngci/tools4cities/middleware/test/TestFireBase.java

```diff
@@ -0,0 +1,67 @@
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.FirestoreOptions;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+import com.google.auth.oauth2.GoogleCredentials;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TestFireBase {
+
+    private static FirebaseDatabase database;
+
+    @BeforeAll
+    public static void setup() throws IOException {
+        FileInputStream serviceAccount =
+                new FileInputStream("./src/test/data/citylayer-middleware.json");
+
+        FirebaseOptions options = new FirebaseOptions.Builder()
```

Comment:
The Builder method is deprecated, can you do some research and try to replace it with a more recent implementation?